.env.example

```
@@ -5,8 +5,10 @@
 # --- Networking ---
 # Port the bridge listens on (loopback only; tailscale serve proxies to it).
 COLLIE_PORT=8787
-# Bind host. Keep this loopback. Binding 0.0.0.0 makes identity checks meaningless.
+# Bind host. Keep this loopback. A non-loopback value refuses to start unless you also set
+# COLLIE_ALLOW_NON_LOOPBACK_BIND=1 (the identity, device and same-origin gates are then client-settable).
 COLLIE_HOST=127.0.0.1
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The COLLIE_HOST key should go before the COLLIE_PORT key

(UnorderedKey)